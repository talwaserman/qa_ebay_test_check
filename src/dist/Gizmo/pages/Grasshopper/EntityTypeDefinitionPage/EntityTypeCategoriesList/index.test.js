"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _regenerator = _interopRequireDefault(require("@babel/runtime/regenerator"));

var _asyncToGenerator2 = _interopRequireDefault(require("@babel/runtime/helpers/asyncToGenerator"));

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _services = require("../services");

var _index = _interopRequireDefault(require("./index"));

it('EntityTypeCategoriesList', /*#__PURE__*/(0, _asyncToGenerator2.default)( /*#__PURE__*/_regenerator.default.mark(function _callee() {
  var tree;
  return _regenerator.default.wrap(function _callee$(_context) {
    while (1) {
      switch (_context.prev = _context.next) {
        case 0:
          tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_index.default, {
            getEtCategories: _services.getEtCategories,
            rowId: '1',
            siteId: 'US'
          })).toJSON();
          expect(tree).toMatchSnapshot();

        case 2:
        case "end":
          return _context.stop();
      }
    }
  }, _callee);
})));