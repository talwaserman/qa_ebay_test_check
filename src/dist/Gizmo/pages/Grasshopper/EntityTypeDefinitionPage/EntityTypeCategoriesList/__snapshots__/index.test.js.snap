// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EntityTypeCategoriesList 1`] = `
<div
  className="et-definition-categorieslist-wrapper"
>
  <br />
  <div
    className="categories-type-header"
  >
    <div
      className="categories-type-label"
    >
      US Category List
    </div>
    <div
      className="categories-count"
    >
      (
      0
      )
    </div>
    <div
      className="categories-type-hint"
    >
      <div
        className="categories-type-icon"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <i
          aria-label="icon: question-circle"
          className="anticon anticon-question-circle"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="question-circle"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
            />
            <path
              d="M623.6 316.7C593.6 290.4 554 276 512 276s-81.6 14.5-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56.1 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.1 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5.1-39.3-17.1-76-48.3-103.3zM472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0z"
            />
          </svg>
        </i>
      </div>
    </div>
  </div>
  <br />
  <div
    className="categories-type-input"
  >
    <input
      className="ant-input"
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Search Categories..."
      type="text"
      value=""
    />
  </div>
  <div
    className="categories-found-list"
  >
    <div
      className="ant-list ant-list-sm ant-list-split"
    >
      <div
        className="ant-spin-nested-loading"
      >
        <div
          className="ant-spin-container"
        >
          <div
            className="ant-list-empty-text"
          >
            <div
              className="ant-empty ant-empty-normal"
            >
              <div
                className="ant-empty-image"
              >
                <svg
                  height="41"
                  viewBox="0 0 64 41"
                  width="64"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                    transform="translate(0 1)"
                  >
                    <ellipse
                      cx="32"
                      cy="33"
                      fill="#F5F5F5"
                      rx="32"
                      ry="7"
                    />
                    <g
                      fillRule="nonzero"
                      stroke="#D9D9D9"
                    >
                      <path
                        d="M55 12.76L44.854 1.258C44.367.474 43.656 0 42.907 0H21.093c-.749 0-1.46.474-1.947 1.257L9 12.761V22h46v-9.24z"
                      />
                      <path
                        d="M41.613 15.931c0-1.605.994-2.93 2.227-2.931H55v18.137C55 33.26 53.68 35 52.05 35h-40.1C10.32 35 9 33.259 9 31.137V13h11.16c1.233 0 2.227 1.323 2.227 2.928v.022c0 1.605 1.005 2.901 2.237 2.901h14.752c1.232 0 2.237-1.308 2.237-2.913v-.007z"
                        fill="#FAFAFA"
                      />
                    </g>
                  </g>
                </svg>
              </div>
              <p
                className="ant-empty-description"
              >
                No Data
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
